kronor_swish/src/main/res/values-sv/strings.xml

```diff
@@ -8,4 +8,14 @@
     <string name="scan_qr">Skanna den här koden din Swish-app</string>
     <string name="generate_qr">Hämtar QR-kod</string>
     <string name="accept_swish_phone">Öppna Swish App på din telefon och acceptera betalningsförfrågan</string>
+    <string name="network_error">Betalningen kunde inte genomföras på grund av nätverksproblem. Kontrollera att din telefon är ansluten till internet och pröva igen.</string>
```

Comment:
Looking at it in context maybe we should switch `pröva` to `försök`, it makes it consistent with the `försök igen`.

I.e.

```
Betalningen kunde inte genomföras på grund av nätverksproblem. Kontrollera att din telefon är ansluten till internet och försök igen.
```